main/realmain.cc

```diff
@@ -240,6 +245,31 @@ void runAutogen(core::Context ctx, options::Options &opts, WorkerPool &workers,
         auto last = unique(mergedClasslist.begin(), mergedClasslist.end());
         opts.print.AutogenClasslist.fmt("{}\n", fmt::join(mergedClasslist.begin(), last, "\n"));
     }
+    if (opts.print.AutogenSubclasses.enabled) {
+        Timer timeit(logger, "autogenSubclassesPrint");
+
+        // Merge the {Parent: Set{Child1, Child2}} maps from each thread
+        map<string, set<string>> mergedSubclasses;
```

Comment:
What about pulling the merging logic into autogen.cc?

Reply:
+1, let's try to not add it to realmain.cc